src/compiler/core.ts

```diff
@@ -2151,6 +2151,10 @@ namespace ts {
         return arg => f(arg) || g(arg);
     }
 
+    export function not<T extends unknown[]>(f: (...args: T) => boolean): (...args: T) => boolean {

```

Comment:
Remove this function since it’s not used anymore ?

Reply:
Hah, I left it because I feel fairly certain I’ll add it again before long, but I guess I’ll just do it again then 😁